src/network-layer/default/__tests__/RelayDefaultNetworkLayer-test.js

```diff
@@ -174,6 +174,38 @@ describe('RelayDefaultNetworkLayer', () => {
       expect(error.source).toEqual(response);
     });
 
+    it('handles errors with 0 column', () => {
+      const response = {
+        errors: [{
+          message: 'Something went wrong.',
+          locations: [{
+            column: 0,
+            line: 1,
+          }],
+        }],
+      };
+
+      expect(fetch).not.toBeCalled();
+      networkLayer.sendMutation(request);
+      expect(fetch).toBeCalled();
```

Comment:
these are tested elsewhere; probably okay to remove and just test the error message